app/models/Post.java

```diff
@@ -50,23 +53,31 @@ public static List<Post> findTaggedWith(String... tags) {
         ).bind("tags", tags).bind("size", tags.length).fetch();
     }
 
+    public Post profileIs(String creators) {
+        profiles.add(Profile.findOrCreateByName(creators));
+        return this;
+    }
+
+    public static List<Post> findMadeBy(String profile) {
+        return Post.find(
+                "select distinct p from Post p join p.profiles as pf where pf.creator = ?1", profile
+        ).fetch();
+    }
     public Post addComment(String author, String content) {
         Comment newComment = new Comment(this, author, content).save();
         this.comments.add(newComment);
         this.save();
         return this;
     }
-
     public Post addLike(String author) {
-        reactLike newLike = new reactLike(this, author).save();
+        ReactLike newLike = new ReactLike(this, author).save();
         this.likes.add(newLike);
         this.save();
         return this;
     }
     public Post previous() {
```

Comment:
Make sure we have some white space between methods. The Intellij code formatter will fix some of this for you. We do follow a style guide so that all code is uniform. https://google.github.io/styleguide/javaguide.html